mgmt/dp/mizar/workflows/nets/delete.py

```diff
@@ -19,9 +21,14 @@ def run(self):
 		logger.info("Run {task}".format(task=self.__class__.__name__))
 		n = nets_opr.store.get_net(self.param.name)
 		n.set_obj_spec(self.param.spec)
+		
+		while len(endpoints_opr.store.get_eps_in_net(n.name)):
+			pass
```

Comment:
Add a todo commit here and refer to the error handling status...

Reply:
Added